.github/CODEOWNERS

```
@@ -0,0 +1,10 @@
+# This is a CODEOWNERS file.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# they will be requested for review when someone opens a pull request.
+/torchtitan @tianyu-l @wconstab @fegin @wwwjn
```

also include the root directory
```suggestion
* @tianyu-l @fegin @wwwjn
```